areas/storage/src/AzureMcp.Storage/Commands/Blob/Batch/BatchSetTierCommand.cs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AzureMcp.Core.Commands;
+using AzureMcp.Core.Services.Telemetry;
+using AzureMcp.Storage.Commands.Blob.Container;
+using AzureMcp.Storage.Options;
+using AzureMcp.Storage.Options.Blob.Batch;
+using AzureMcp.Storage.Services;
+using Microsoft.Extensions.Logging;
+
+namespace AzureMcp.Storage.Commands.Blob.Batch;
+
+public sealed class BatchSetTierCommand(ILogger<BatchSetTierCommand> logger) : BaseContainerCommand<BatchSetTierOptions>()
+{
+    private const string CommandTitle = "Set Access Tier for Multiple Blobs";
+    private readonly ILogger<BatchSetTierCommand> _logger = logger;
+
+    private readonly Option<string> _tierOption = StorageOptionDefinitions.Tier;
+    private readonly Option<string[]> _blobNamesOption = StorageOptionDefinitions.BlobNames;
+
+    public override string Name => "settier";
```

Comment:
I know the agent will be the one calling this but "settier" is hard to read as "Set Tier" at first glance.  Anyway to make this easier to read?  Is there any other prior art for something like "set_tier"

Reply:
I see DataLake is using `list-paths`, will update to `set-tier`.